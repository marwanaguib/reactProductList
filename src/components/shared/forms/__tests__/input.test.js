import React from "react";
import { shallow } from "enzyme";

import Input from "../input";

const props = {
  func: jest.fn(),
  name: "test",
  elemKey: "code",
  type: "text",
};

describe("inputForm", () => {
  const componetWrapper = shallow(<Input {...props} />);
  it("render inputForm to match snapshot", () => {
    expect(componetWrapper.find(`input[type="${props.type}"]`).length).toBe(1);
    expect(componetWrapper).toMatchSnapshot();
  });
});
