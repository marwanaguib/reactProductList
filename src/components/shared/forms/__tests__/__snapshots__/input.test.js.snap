// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inputForm render inputForm to match snapshot 1`] = `
<styled.form
  onSubmit={[Function]}
>
  <input
    placeholder="test"
    type="text"
  />
  <input
    type="submit"
    value="Search"
  />
</styled.form>
`;
